app/components/containerInspect/containerInspectController.js

```diff
@@ -0,0 +1,18 @@
+angular.module('containerInspect', [])
+.controller('ContainerInspectController', ['$scope', '$transition$', 'Notifications', 'ContainerService',
+function ($scope, $transition$, Notifications, ContainerService) {
+  $scope.state = {};
+  $scope.state.loaded = false;
```

Comment:
No need to create a `$scope.state` object, please use the standard `initView()` function here. Have a look at https://github.com/portainer/portainer/blob/develop/app/components/containerStats/containerStatsController.js#L146-L157 for an example of query/then/catch/finally implementation.